public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -78,16 +79,16 @@ export class TableContainer extends PureComponent<Props> {
     return (
       <>
         {frames && frames.length === 0 && (
-          <PanelChrome title={'Table'} width={width} height={200}>
-            {() => <MetaInfoText metaItems={[{ value: '0 series returned' }]} />}
+          <PanelChrome title={t('table-container-title', 'Table')} width={width} height={200}>
```

Comment:
Please check [i18n guidelines](https://github.com/grafana/grafana/blob/813ebf988d354bf23e981e00c9c8458995774c21/contribute/internationalization.md#phrase-id-naming-convention). We should use 2 or 3 level message IDs depending on the context. In this case I think something like `explore.table.title` should work.